lib/dotcom_web/components/svg.ex

```diff
@@ -0,0 +1,38 @@
+defmodule DotcomWeb.Components.Svg do
+  use Phoenix.Component
+
+  attr :rest, :global
+
+  def logan_express_bb_icon(assigns) do
+    ~H"""
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 8.5 8.5" {@rest}>
+      <circle fill="#f16823" cx="4.25" cy="4.25" r="4.25" />
+      <path
+        fill="#b74326"
+        d="M6.95,6.47h0c-.65.79-1.62,1.28-2.7,1.28-1.93,0-3.5-1.57-3.5-3.5,0-.35.05-.69.15-1.01.28-.04.59-.01.92.1.61.25,1.02.68,1.02,1.14.36-.34,1-.43,1.61-.2.78.28,1.25.98,1.08,1.59.25-.14.53-.14.79-.06.31.12.52.36.63.66Z"
+      />
+      <path
+        fill="#fff"
+        d="M6.04,1.63c.15-.07.48-.07.55.08s-.13.41-.28.49l-.52.25-.07,1c0,.06-.04.11-.09.13l-.25.12s-.1,0-.11-.06l-.15-.87-.46.22-.07.35s-.02.05-.04.06l-.19.09s-.08,0-.09-.03c0,0,0-.01,0-.02l-.1-.56-.38-.42s0-.01-.01-.02c-.02-.03,0-.08.03-.09l.19-.09s.05-.01.07,0l.32.16.46-.22-.59-.66s-.02-.1.02-.12l.25-.12c.05-.02.11-.02.16.01l.82.57.52-.25h0Z"
+      />
+      <path
+        fill="#fff"
+        d="M6.95,6.47h0c-.65.79-1.62,1.28-2.7,1.28-1.93,0-3.5-1.57-3.5-3.5,0-.35.05-.69.15-1.01.28-.04.59-.01.92.1.61.25,1.02.68,1.02,1.14.36-.34,1-.43,1.61-.2.78.28,1.25.98,1.08,1.59.25-.14.53-.14.79-.06.31.12.52.36.63.66Z"
+      />
+    </svg>
+    """
+  end
+
+  attr :rest, :global
+
+  def walk_icon(assigns) do
+    ~H"""
+    <svg role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" {@rest}>
+      <g fill="#1C1E23" fill-rule="nonzero" transform="matrix(-1 0 0 1 14 0)">
+        <circle cx="4.747" cy="1.318" r="1.318" />
+        <path d="M.602 7.086c.366-.064 1.833-.318 1.915-.345.093-.031 0 0 .03-.047.032-.047 1.311-2.745 1.436-2.886.125-.14.468-.499.951-.655.484-.156.967.094.967.094S8.304 4.62 8.46 4.697c.157.079.297.36.297.36l1.121 2.108a.678.678 0 1 1-1.191.579L7.641 5.766 6.985 5.4c.047.14.648 3.065.648 3.065l.608 3.463 2.132 2.568a.783.783 0 1 1-1.17 1.025l-2.545-2.993c-.21-.25-.265-.375-.296-.538-.031-.164-.383-2.278-.383-2.278l-2.012 2.013v3.548h-.004a.788.788 0 0 1-1.567 0h-.004v-.068-.012-.016c.004-.54.02-3.233 0-3.842-.024-.679.218-.827.273-.89.055-.062 1.934-2.058 1.934-2.058l-.413-2.004s-.538 1.052-.593 1.17c-.055.116-.226.327-.405.366-.18.039-2.418.491-2.418.491l-.001-.004c-.033.004-.066.01-.1.01a.666.666 0 1 1-.066-1.33z" />
+      </g>
+    </svg>
+    """
+  end
```

Comment:
I would put these svgs into files and then use something like: https://github.com/anthonyshull/mbta_metro/blob/main/lib/mbta_metro/components/icon.ex to compile them when needed. I still think they would best go in a separate library.